src/tribol/geom/ContactPlane.cpp

```diff
@@ -29,9 +29,9 @@ namespace tribol
 FaceGeomError CheckInterfacePair( InterfacePair& pair,
                                   ContactMethod const cMethod,
                                   ContactCase const TRIBOL_UNUSED_PARAM(cCase),
-                                  bool& inContact )
+                                  bool& isInteracting )
```

Comment:
I think the header needs to be updated too.  Also, a definition of an interacting face pair would be helpful.